source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,588 @@
+---
+title: ovirt-provider-ovn security groups
+category: feature
+authors: mdbarroso
+feature_name: security-group-support
+feature_modules: engine,ovirt-provider-ovn
+feature_status: In Development
+---
+
+
+# Ovirt-provider-ovn security groups
+
+
+## Summary
+
+This feature aims to implement the OpenStack Networking API [security group](https://developer.openstack.org/api-ref/network/v2/index.html#security-groups-security-groups)
+in the ovirt-provider-ovn.
+
+By doing so, we allow fine-grained access control to - and from - the oVirt
+VMs attached to external networks.
+
+The Networking API v2 defines security groups as a white list of rules.
+That means, that by default, neither incoming nor outgoing traffic is
+allowed (from the VMs perspective).
+
+A security group is applied to a logical port - a *logical switch port*
+in the OVN model.
+
+### Owner
+
+* Feature Owner: [Miguel Duarte Barroso](https://github.com/maiqueb) (mbarroso on IRC)
+* Email: <mdbarroso@redhat.com>
+
+### Terminology
+
+* Networking API: [OpenStack RESTful Networking API](https://developer.openstack.org/api-ref/network/v2/)
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that
+brings Software Defined Networking to OVS.
+* oVirt OVN provider: a proxy that implements a subset of the Networking API
+and interacts with OVN.
+* Security group: container for security group **rules**; networking API
+entity
+* Security group rules: specify the access to a network resource;
+networking API entity
+
+### Benefit to oVirt
+
+Currently, the only security mechanisms present for oVirt VMs attached to
+external networks are quite crude L2 and L3 protections - MAC
+spoofing, and IP spoofing - implemented directly by Open vSwitch. It is worth
+mentioning that IP spoofing protection is only applied when a subnet is defined
+on top of the Networking API network.
+
+Security groups, which are a traffic white list, complement provided security
+by specifying which traffic is allowed to *and* from the resources - e.g. ports - using high level abstraction L3 and L4 semantics.
+
+Furthermore, security groups allow these rules to be applied in fine
+grained fashion - e.g. the user can specify which rules apply to which
+ports.
+
+Below, you can find some examples of goals for this feature:
+* **only** allow incoming traffic to a specific VM from a specific
```

Comment:
Is the "only" necessary?

Reply:
hm, I think so; it we don't implement this feature, 'we allow incoming traffic to a specific VM'. 

We want to allow when a set of conditions are met, and reject everything else.